### ISSUE TITLE
Cita preferences not translated to some languages

### ISSUE THREAD

**Issue by diegodlh** [diegodlh/zotero-cita#288]
Cita preferences are not translated to most of the languages we support.

Apparently they are only translated to English (`en-US`) and German (`de`), and all other translations are ignored.

Because both `en-US` and `de` are supported locales by Zotero, I guess this may be related to the way how we support locales not supported by Zotero (see 15de93b). Note, however, that they are not translated to `zh-Hant` and `zh-Hans`, even though these are supported by Zotero, and we have preferences translations for these languages.

Maybe this could be fixed if we move that piece of code from `src/utils/locale.ts` to `static/bootstrap.js`, as suggested [here](https://github.com/diegodlh/zotero-cita/blob/15de93bed3e8412e0eea3e3023b21731fe4af174/src/utils/locale.ts#L42)?

The thread I opened in the zotero-dev mailing list may provide further ideas: https://groups.google.com/g/zotero-dev/c/GUovOSrtQy4

**Comment by diegodlh** [comment]
Note that translations do work in `citationEditor.xhtml`, `citationImporter.xhtlm` and `identifierImporter.xhtml` (have not tried `selector.xhtml`).

However, these use `Wikicite.getString()` (which in turn relies on our custom `Localization` object set up by our `initLocale` function), as opposed to `preferences.xhtml` which relies on native Fluent [markup-based approach](https://www.zotero.org/support/dev/zotero_7_for_developers#replacing_dtd_files).

**Comment by diegodlh** [comment]
Noting too that `Zotero.plugins.registerLocales()` seems to be running *before* the extension bootstrap.js' `startup()` function. So moving the aforementioned code into `bootstrap.js` will probably *not* fix this issue.

**Comment by diegodlh** [comment]
7033ea2 fixed that the item pane section header label and sidenav tooltiptext did not show.

However, they also seem to be not translated to languages other than `en-US` and `de`.

**Comment by thebluepotato** [comment]
Just to clarify what's left to do here, is it only the tooltips now? Because preference panes and sidenav section header at least seem to follow the correct translations

**Comment by Dominic-DallOsto** [comment]
Yeah, after your latest commit the tooltips and pane headers all seemed to translate successfully for me
